config/routes.rb

```diff
@@ -0,0 +1,13 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  root "pages#show", page: "home"
+
+  resources :passengers, :drivers
+
+  resources :trips, except: [:index, :edit]
+
+  get "/pages/:page" => "pages#show"
```

Comment:
Interesting, especially in conjunction with the implementation of the show action for PagesController... Though I'm not convinced that it needed to be like this-- why not just have a specific route for a "home" action defined in the PagesController, instead of using a URL param?

aka
```ruby
get "/home" => "pages#home"
```

Reply:
It was set up so that many static pages could be created by any name. that obv wasn't used here though!